### PR TITLE
[Data Sources] Add: ATSD query runner

### PR DESCRIPTION
**Pull request by rmakulov**
Related to [following discussion](https://github.com/getredash/redash/issues/1579).
I will publish more detailed instructions on how to use this later on (will post a comment with the link).

### CODE REVIEW

**Comment by rmakulov** [comment]
[Detailed instructions](https://github.com/getredash/website/pull/39)

File: redash/query_runner/axibase_tsd.py [hunk 1]
```diff
@@ -0,0 +1,191 @@
+from io import StringIO
+import json
+import logging
+import sys
+import uuid
+
+import pandas as pd
+import atsd_client
+from atsd_client.exceptions import SQLException
+from atsd_client.services import SQLService, MetricsService
```
Comment (arikfr) [inline redash/query_runner/axibase_tsd.py]: All the ATSD imports should be in a try/except block along with an enabled flag. 

See [here](https://github.com/getredash/redash/blob/master/redash/query_runner/mssql.py#L11-L15) (and [here](https://github.com/getredash/redash/blob/master/redash/query_runner/mssql.py#L75-L77)) for example.

File: redash/query_runner/axibase_tsd.py [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+    """
+    meta, data = csv_response.split('\n', 1)
+    meta = meta[1:]
+
+    meta_with_padding = meta + '=' * (4 - len(meta) % 4)
+    meta_decoded = meta_with_padding.decode('base64')
+    meta_json = json.loads(meta_decoded)
+    meta_columns = meta_json['tableSchema']['columns']
+
+    df = pd.read_csv(StringIO(data), sep=',')
```
Comment (arikfr) [inline redash/query_runner/axibase_tsd.py]: It's a bit too much to include Pandas just to parse CSV. Why not use the builtin `csv` module?

File: redash/settings.py [hunk 3]
```diff
@@ -241,7 +242,7 @@ def all_settings():
     'showPermissionsControl': FEATURE_SHOW_PERMISSIONS_CONTROL,
     'allowCustomJSVisualizations': FEATURE_ALLOW_CUSTOM_JS_VISUALIZATIONS,
     'dateFormat': DATE_FORMAT,
-    'dateTimeFormat': "{0} HH:mm".format(DATE_FORMAT),
+    'dateTimeFormat': "{0} HH:mm:ss".format(DATE_FORMAT),
```
Comment (arikfr) [inline redash/settings.py]: Let's keep it as it was to avoid surprises for existing deployments.

File: redash/query_runner/axibase_tsd.py (hunk 2 above)
Comment (arikfr) [inline redash/query_runner/axibase_tsd.py]: I also noticed that Pandas is a dependency of `atsd_client` -- is it really mandatory?

File: requirements.txt [hunk 4]
```diff
@@ -28,6 +28,7 @@ SQLAlchemy==1.1.4
 sqlparse==0.1.8
 wsgiref==0.1.2
 honcho==0.5.0
+atsd_client==2.0.9
```
Comment (arikfr) [inline requirements.txt]: This belongs to `requirements_all_ds.txt`.

File: redash/query_runner/axibase_tsd.py (hunk 2 above)
Comment (rmakulov) [inline redash/query_runner/axibase_tsd.py]: Ok, no.

**Comment by rmakulov** [comment]
@arikfr is there anything to be done on this PR?

**Comment by arikfr** [comment]
Merged :) thanks!

**Pull request merged**